### PR TITLE
fix: Restore dropdown support for multiple triggers

### PR DESCRIPTION
**Pull request by crowjonah**
We have a dropdown (in acadia/field maps) that can be expanded by focusing an input or clicking an icon. This worked as of `v1.0.0-beta.25`, and this PR restores that behavior.

It's slightly more cumbersome to track multiple triggers, especially when handling click events, and it raises the question of which trigger to restore focus to upon close (I arbitrarily chose the one at index 0).

First-time contributor here, so let me know if I've missed any important part of the PR process. It looks like the repo's `prettier` settings are creating some noise in the files I touched, but I could be wrong about that.

### CODE REVIEW

File: src/demos/calcite-dropdown.html [hunk 1]
```diff
@@ -308,6 +308,17 @@ <h3>Not calcite-button triggers</h3>
         <calcite-dropdown-item>Table</calcite-dropdown-item>
       </calcite-dropdown-group>
     </calcite-dropdown>
+    <br />
+
+    <calcite-dropdown>
+      <input slot="dropdown-trigger" placeholder="I'm an input dropdown-trigger, and the carat is too!" />
```
Comment (paulcpederson) [inline src/demos/calcite-dropdown.html:314]: Suuuper picky, but have a little misspelling here.
```suggestion
      <input slot="dropdown-trigger" placeholder="I'm an input dropdown-trigger, and the caret is too!" />
```

**Comment by paulcpederson** [comment]
Nice, thanks @crowjonah 👍

File: src/demos/calcite-dropdown.html (hunk 1 above)
Comment (crowjonah) [reply src/demos/calcite-dropdown.html:314]: Funny, I initially misspelled it as the icon's attribute value and fixed it there, forgetting I'd written it here too. Thanks!

**Comment by macandcheese** [comment]
Looks good ! 👍

**Pull request merged**